.eslines.json

```diff
@@ -0,0 +1,9 @@
+{
+    "rulesNotToDowngrade": ["no-unused-vars"],
+    "remote": "origin/staging",
+    "processors": {
+	    "default": "lines-modified",
+	    "staging": "parsing-errors",
```

Comment:
as far as the branch names go in github, this should be not necessary for happychat-service